apisix/core/config_local.lua

```diff
@@ -65,12 +63,6 @@ function _M.local_conf(force)
         return nil, err
     end
 
-    -- fill the default value by the schema
```

Comment:
why remove this?

Reply:
@nic-6443 

https://github.com/apache/apisix/pull/11993#discussion_r1982986021
https://github.com/apache/apisix/pull/11993#discussion_r1982994653

We have a two guarantees, so this is not necessary.

Reply:
This has been moved to read_yaml_conf. Please check description for reason